test/basic.jl

```diff
@@ -419,3 +419,18 @@ end
         @test all(nf1 .== nf2)
     end
 end
+
+
+@testset "AbstractSystem -> Molly System" begin
+    system = make_test_system().system;
+    #Update values to be something that works with Molly
+    system = AbstractSystem(system; 
+                boundary_conditions = [Periodic(), Periodic(), Periodic()],
+                bounding_box = [[1.54732, 0.0, 0.0],
+                                [0.0, 1.4654985, 0.0],
+                                [0.0, 0.0, 1.7928950]]u"Å");
+    molly_sys = System(system)
+    test_approx_eq(system, molly_sys; common_only = true)
```

Comment:
Sorry, only got round to look at this now. This is not the best way to do this. You can directly overwrite things (or drop keys) from `make_test_system` and then one should be able to use `test_approx_eq` directly without using the `common_only=true` flag.